.kitchen.yml

```diff
@@ -47,14 +48,28 @@ platforms:
 - name: high-sierra-chef13
   driver:
     box: microsoft/macos-high-sierra
-    version: 10.13.5
+    version: 10.13.6-v2
   provisioner:
     product_version: 13
 
 - name: high-sierra-chef14
   driver:
     box: microsoft/macos-high-sierra
-    version: 10.13.5
+    version: 10.13.6-v2
+  provisioner:
+    product_version: 14
+
+- name: mojave-chef13
+  driver:
+    box: microsoft/macos-mojave
+    version: 10.14.0
```

Comment:
`box_check_update: true` is the default

Reply:
No it isn't: https://docs.chef.io/config_yml_kitchen.html

Reply:
Ah - I was thinking of Vagrant’s default: https://www.vagrantup.com/docs/boxes/versioning.html

Either way, Vagrant’s docs indicate that this option is just a “check”, which is why we pinned these to force an update.

Reply:
I'm aware of why we pinned them. What I'm saying is that I don't think it's helpful anymore since we can make base boxes more efficiently now. We haven't developed a system to update the pin in every cookbook every time there's a new base box "released". We've had newer base boxes for a while, and this change is just now coming...meaning that we've been building on a much older version of the High Sierra box, for example.

We'll make a quick change to our build cookbook to ensure that our build nodes have the latest versions of the boxes and nothing else, instead of trying to align version pinning in all of our `.kitchen.yml` files. In other words, implement `vagrant box prune --force` and `vagrant box update --box BOX --force` to manage the boxes that are on our build machines.

Reply:
Updating the build machine cookbook seems like a good plan. Will adding `box_check_update: true` do anything for us here?